server/CMakeLists.txt

```diff
@@ -13,7 +13,6 @@ include_directories(${CMAKE_SOURCE_DIR}/external_libraries
 					${CMAKE_SOURCE_DIR}/external_libraries/nova-tt
 					)
 
-include_directories(${boost_include_dirs})
```

Comment:
We are sure this doesn't impact e.g. the external plugin interface?

Reply:
Do internal plugins access the server via the same plugin api? If this is the case then yes :) Otherwise it also probably works ;)